functions/exercise4/script.js

```diff
@@ -1 +1,11 @@
-"use strict";
\ No newline at end of file
+"use strict";
+
+function validateAsPhraseConsistsOfLettersOnly(phrase) {
+    var validate = /^[a-zA-Z]*$/g.test(phrase);
```

Comment:
Do you need this variable validate? Why can't you simply put assigned value into if condition in line 8 ?